cves/kernel/CVE-2012-6544.yml

```diff
@@ -332,8 +321,8 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: true
+  note: Commited by a different person then wrote it requiring vouching for the fix.
```

Comment:
Fix up the language here. "Committed by a different person than who wrote the bug. This required another developer to vouch for the fix" - Or something similar.